README.md

```diff
@@ -88,14 +88,21 @@ and select the one that is of interest.
 python metriq_gym/run.py poll
 ```
 
-### Listing jobs
+### View jobs
 
 You can view all the jobs that have been dispatched by using the `list-jobs` action. This will read the `jobs_file`
-and display information about each job, including its ID, backend, job type, provider, qubits, and shots.
+and display basic information about each job, including its ID, backend, job type, provider, and device.
 
 ```sh
 python metriq_gym/run.py list-jobs
```

Comment:
perhaps `list-jobs` and `view` actions should be consolidated in one 🤔

Reply:
Note that `view` without any argument shows the full list anyway. Maybe that's enough as a replacement for `list-jobs`. What do you think?